.qlty/qlty.toml

```diff
@@ -0,0 +1,109 @@
+# This file was automatically generated by `qlty init`.
+# You can modify it to suit your needs.
+# We recommend you to commit this file to your repository.
+#
+# This configuration is used by both Qlty CLI and Qlty Cloud.
+#
+#     Qlty CLI -- Code quality toolkit for developers
+#     Qlty Cloud -- Fully automated Code Health Platform
+#
+# Try Qlty Cloud: https://qlty.sh
+#
+# For a guide to configuration, visit https://qlty.sh/d/config
+# Or for a full reference, visit https://qlty.sh/d/qlty-toml
+config_version = "0"
+
+exclude_patterns = [
+  "*_min.*",
+  "*-min.*",
+  "*.min.*",
+  "**/.yarn/**",
+  "**/build/**",
+  "**/cache/**",
+  "**/config/**",
+  "**/db/**",
+  "**/deps/**",
+  "**/dist/**",
+  "**/extern/**",
+  "**/external/**",
+  "**/generated/**",
+  "**/seed/**",
+  "**/target/**",
+  "**/testdata/**",
+  "**/node_modules/**",
+  "**/vendor/**",
+  "app/assets/images/**/*",
+  "app/javascript",
```

Comment:
This one also we don't want checked?

Reply:
We could put it back in. Codeclimate was not good for typescript and we already checked it via eslint. We'll take a look at it.

Reply:
eslint is also fine in case qlty brings no improvement, makes sense whatever you decide